src/Illuminate/Http/Client/Response.php

```diff
@@ -277,17 +278,16 @@ public function toException()
     /**
      * Throw an exception if a server or client error occurred.
      *
+     * @param  \Closure|null  $callback
      * @return $this
      *
      * @throws \Illuminate\Http\Client\RequestException
      */
-    public function throw()
+    public function throw(?Closure $callback = null)
```

Comment:
I'd use `?callable` instead of `?Closure` to allow for invokable objects, and other callable (such as denoted by the array syntax).

```php
public function throw(?callable $callback = null)
```

P.S.: don't forget to update the docblock

Reply:
Thanks for your help.